tool/release_note_helper.dart

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env dart
+
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:args/command_runner.dart';
+import 'lib/release_note_classes.dart';
+
+void main(List<String> args) {
+  final runner = CommandRunner(
+    'release_note_helper',
+    'Helps manage version notes for release.',
+  )
+    ..addCommand(VerifyCommand())
+    ..addCommand(MarkDownCommand())
+    ..addCommand(BackfillPullRequestUrlCommand());
+
+  runner.run(args).catchError((error) {
+    if (error is! UsageException) throw error;
+    print(error);
+    exit(64); // Exit code 64 indicates a usage error.
+  });
+  return;
+}
+
+class MarkDownCommand extends Command {
+  @override
+  final name = 'markdown';
+  @override
+  final description = 'Prints all versions listed in the `file`, in markdown.';
```

Comment:
I'm still of the opinion that we don't need to store previous versions of release notes in our repo - these are all stored on the flutter website.